.circleci/config.yml

```diff
@@ -861,4 +861,4 @@ executors:
     environment:
       JVM_OPTS: -Xmx3200m
       BUILDTYPE: Debug
-      GRADLE_OPTS: -Xmx4096m -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process
\ No newline at end of file
+      GRADLE_OPTS: -Xmx4096m
```

Comment:
ok.

Would you recommend that I close this change or that I update the `scripts/android/publish_android.sh` to use `--no-daemon` ?

Reply:
actually it is already there: https://github.com/mapbox/mapbox-maps-android/blob/main/Makefile#L59

Reply:
yeah, I've added that because we sometimes need to use the script manually outside of CI.
I guess we can remove the --no-daemons here and just add this option when it causes flakiness to other tasks.

Reply:
Also I assume it's commonly recommended way to run gradle CI to avoid memory issues, e.g. https://support.circleci.com/hc/en-us/articles/360021812453-Common-Android-memory-issues#gradle

Reply:
I am not sure I agree. The sentence:
> This is not needed in continuous integration since every compilation is fresh

doesn't apply well to us since we have jobs that are not fresh but rather benefit from having loaded data in memory.

When I tested moving the Kotlin compilation to its own daemon the time difference was minor but there was some. It seems that some tasks can run in parallel.

Reply:
Afair problem is often with `daemon has disappeared` due to memory issues. 
Agree on the compilation speed though, hard to say if daemon will work smoothly, we can try and see.